File: kotlin.Ver/app/src/main/java/com/exercise/ckaiself/kaiyan_kotlinver/mvp/model/bean/Category.kt
package com.exercise.ckaiself.kaiyan_kotlinver.mvp.model.bean

import java.io.Serializable

/**
 * @author Ckai
 * @date 2019/1/11 5:15 PM
 * @description
 */

data class Category(
        val id: Long,

        val name: String,

        val description: String,

        val bgPicture: String,

        val bgColor: String,

        val headerImage: String
) : Serializable

File: flutterversion/lib/bean/Category.dart
import 'package:json_annotation/json_annotation.dart';
/*
 * @Created Date: 2019-02-07 11:06
 * @Author: Ckai
 * @Description: 
 */
part 'Category.g.dart';

@JsonSerializable()
class Category {
  num id;
  String name;
  String description;
  String bgPicture;
  String bgColor;
  String headerImage;

  Category({
    this.id,
    this.name,
    this.description,
    this.bgPicture,
    this.bgColor,
    this.headerImage,
  });

  factory Category.fromJson(Map<String, dynamic> json) =>
      _$CategoryFromJson(json);
}
